app/flask_config.py

```
@@ -61,7 +63,21 @@ class TestConfig(BaseConfig):
 
     TESTING = True
     DEBUG = True
-    SQLALCHEMY_DATABASE_URI = "postgresql://postgres@localhost/noharm"
+    CORS_ORIGINS = [Config.MAIL_HOST, "http://localhost:3000"]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)